android-json-form-wizard/src/main/java/com/vijay/jsonwizard/fragments/JsonWizardFormFragment.java

```diff
@@ -149,14 +148,12 @@ private Form getForm() {
     @Override
     public void onResume() {
         super.onResume();
+        processSkipSteps();
```

Comment:
Rename this to Process Previous step since the method functionality has been changed